cli.sh

```
@@ -1113,30 +1113,165 @@ copy_claude_configs() {
+		command=$(jq -r ".mcpServers[\"$server_name\"].command" "$registry_file")
+		args=$(jq -r ".mcpServers[\"$server_name\"].args | @sh" "$registry_file")
+		requires=$(jq -r ".mcpServers[\"$server_name\"].requires | @sh" "$registry_file")
+		category=$(jq -r ".mcpServers[\"$server_name\"].category" "$registry_file")
+
+		# Check if prerequisites are met
+		local prereqs_met=true
+		local missing_prereqs=()
+		if [ -n "$requires" ] && [ "$requires" != "''" ]; then
+			# Parse the shell-escaped array
+			eval "local prereq_array=$requires"
+			for prereq in "${prereq_array[@]}"; do
```

Shellcheck (0.11.0)

[warning] prereq_array is referenced but not assigned.

(SC2154)